app/models/miq_worker.rb

```diff
@@ -514,7 +514,7 @@ def minimal_class_name
 
   def database_application_name
     zone = MiqServer.my_server.zone
-    "MIQ #{Process.pid} #{minimal_class_name}[#{compressed_id}], s[#{miq_server.compressed_id}], #{zone.name}[#{zone.compressed_id}]".truncate(64)
+    "MIQ|#{Process.pid}|#{miq_server.compressed_id}|#{compressed_id}|#{zone.compressed_id}|#{minimal_class_name}|#{zone.name}".truncate(64)
   end
 
```

Comment:
Oh. I understand your point. Good idea. It's will make
MIQ|pid|whatever|a very very very long application name|region_name|
to
`MIQ|pid|whatever|a very very very long application n...|`
not
`MIQ|pid|whatever|a very very very long application n...`
I tested this:
```
irb(main):028:0> a="a|bb|ccc".split("|")
=> ["a", "bb", "ccc"]
irb(main):029:0> a="a|bb|ccc".split("|")[6]
=> nil
```
Ruby won't throw an error but just return nil, so it's fine to lost last `|`. What do you think